### ISSUE TITLE
[announce] `{{changelogContributors}}`, `{{changelogChanges}}` not rendered in smtp

### ISSUE THREAD

**Issue by dhinojosa** [jreleaser/jreleaser#1449]
### Task List

- [x] Steps to reproduce provided
- [x] Stacktrace (if present) provided
- [x] Example that reproduces the problem (link to git repository is ideal)
- [x] Full description of the issue provided (see below)

### Steps to Reproduce

1. Create a jreleaser yaml in a Java project
2. Create a mail template, in _src/jreleaser/templates_, mine is called _mail.tpl_
3. Create a message like the one below...
```tpl
Dearest User,

Here is the latest release of {{projectName}} version {{projectVersion}}!

Some of the latest features for the release include:

{{changelogChanges}}

We would like to thank the following contributors for this latest release.

{{changelogContributors}}
```
4. Run `full-release`

### Expected Behaviour

`{{changelogChanges}}` should render the changelog changes
`{{changelogContributors}}` should render the contributors

### Actual Behaviour

Blank

### Environment Information

- **Operating System**: Apple M1 Max, Ventura 13.4.1
- **JReleaser Version:** 1.7.0
- **JDK Version:** 17.0.6

Please paste stacktraces from `[out|build|target]/jreleaser/trace.log`.

```log

[INFO]    [smtp] subject: New Release of JReleaser Topic Available
[DEBUG]   [smtp] message: Dearest User,
Here is the latest release of jreleaser-demo-yml version 0.0.3!

Some of the latest features for the release include:

We would like to thank the following contributors for this latest release


Cordially,
Our Team

2023-07-14

**Comment by aalmiray** [comment]
**Cannot reproduce**

Given a generated changelog such as 

```
$ cat out/jreleaser/release/CHANGELOG.md 
## Changelog

## 🛠  Build
- 18ed199 Add instructions for Java modules
- 3dc890e Add GH workflows
- 21eb278 Add sources

## 📝 Documentation
- 8b592aa Update readme [skip ci]
- 8be3888 Add readme [skip ci]

---
- ffdecbc Init repository


## Contributors
We'd like to thank the following people for their contributions:
- Andres Almiray
```

With a "dummy" smtp announcer configured as

```yaml
announce:
  smtp:
    active: ALWAYS
    host: smtp.acme.com
    username: foo
    from: user@acme.com
    to: user@acme.com
```

Running on dryrun and debug yields

```
[INFO]    [smtp] subject: Helloworld 1.0.0-SNAPSHOT released!
[DEBUG]   [smtp] message: Dearest User,

Here is the latest release of helloworld version 1.0.0-SNAPSHOT!

Some of the latest features for the release include:

## 🛠  Build
- 18ed199 Add instructions for Java modules
- 3dc890e Add GH workflows
- 21eb278 Add sources

## 📝 Documentation
- 8b592aa Update readme [skip ci]
- 8be3888 Add readme [skip ci]

---
- ffdecbc Init repository



We would like to thank the following contributors for this latest release.

## Contributors
We'd like to thank the following people for their contributions:
- Andres Almiray
[INFO]    [smtp] Sending e-mail announcement
```

**Comment by aalmiray** [comment]
🎉 This issue has been resolved in `v1.8.0` ([Release Notes](https://github.com/jreleaser/jreleaser/releases/tag/v1.8.0))
